lang.cpp

```diff
@@ -45,39 +47,48 @@ int main(){
         word += ch;
       }
   }
-  //if(word != ""){
-    //code_list.push_back(word);
-  //}
+
 
   code_list.pop_back();
   code_list.pop_back();
 
-//  for(int i = 0; i < code_list.size(); i++){
-  //  cout << code_list[i] << endl;
-  //}// operability check
+/*  <=== Operability Check of Parser === >
+ for(int i = 0; i < code_list.size(); i++){
+   cout << code_list[i] << endl;
+  }
+*/
 
   vector<string> transfer;
-
   string id = "Identifier";
```

Comment:
You don't need these variables, because You can push constants:
`transfer[i].push_back("Literal")`